Changelog

```diff
@@ -1,3 +1,67 @@
+Version 3.1, 2019-09-XX
+
+  Features:
+  * Allow user attributes in policy conditions (#1645)
+  * Assign tokens and set old PIN during migration (#1619)
+
+  Enhancements:
+  * Authentication and Challenge Response:
+    * RADIUS token supports a single AccessChallenge with the remote RADIUS server (#1790)
+    * Improving Push token performance by reusing still valid access token (#1795)
+    * Improving TiQR token: It returns the remaining attemps after a wrong PIN is given (#1777)
+    * Enhance validate check logic in regards to serials and user names (#1768)
+    * User may now have several TiQR tokens at the same time (#1739)
+    * Do not increase fail counter when *checking* for an answered challenge (#1697)
+    * Allow additional token specific checks when answering challenge response (#1695)
+    * Endpoint GET /token/challenges also takes transaction_id (#1689)
+    * Push token can delay the response of /validate/check, so that there is no need 
+      to query the server to check if the push notification has been answered (#1583)
+  * User experience:
+    * Improve user experience when enrolling Yubikeys via ykpersonalize - Automatically 
+      removing whitespaces (#1735)
+    * Allow user to change the token description (#1717)
+    * Customize Web UI page title (#1624, #1243)
+    * *search_on_enter* also applies to audit log (#1493)
+    * Allow a welcome message in the Web UI if the user has not token (#1074)
+  * Management:
+    * Add script to export OTP counters (#1728)
+    * Admins can only see tokens within the realm they are allowed to manage (#1713)
+    * Allow many additional tags in email notifications: serial, user, givenname,
+      surname, username, userrealm, tokentype, recipient_givenname, recipient_surname,
+      time, date (#1703)
+    * Improve diagnostics script by adding SQLAlchemy URL (#1667)
+    * Add resolver conditions to several policy checks (#1646)
+    * /auth entries in the audit log now also fill in resolver and serial (#1593)
+    * `pi-manage backup` also backs up the FreeRADIUS configuration (#1575)
+    * Allow event handlers on /auth endpoint (#1567)
+    * Add adminread policies for policies, events, resolvers, system, machineresolvers,
```

Comment:
I think this could also be a main feature.

Reply:
done in  7d9d519c